cli/cmd/addon_publish.go

```diff
@@ -167,12 +168,17 @@ func createNewAddonDraftVersion(ctx context.Context, c *cloudquery_api.ClientWit
 		body.Doc = string(b)
 	}
 
-	if manifest.PathToMessage != "" {
-		b, err := os.ReadFile(filepath.Join(manifestDir, manifest.PathToMessage))
-		if err != nil {
-			return fmt.Errorf("failed to read message file: %w", err)
+	if manifest.Message != "" {
```

Comment:
Isn't `message` required? We can add a validation in a separate PR if we don't have it already

Reply:
@erezrokah it's required, but the remote endpoint checks for it anyway. I don't think we need to duplicate all of that in client code.